.github/workflows/dm_binlog_999999.yaml

```diff
@@ -90,8 +105,10 @@ jobs:
       - name: Run test cases
         run: |
           RESET_MASTER=false make dm_integration_test CASE=incremental_mode
-          echo "show binary logs;" | mysql -uroot -h127.0.0.1 -P3306 -p123456 | grep -q "mysql-bin.1000000"
-          echo "show binary logs;" | mysql -uroot -h127.0.0.1 -P3307 -p123456 | grep -q "mysql-bin.1000000"
+          echo "show binary logs;" | mysql -uroot -h127.0.0.1 -P3306 -p123456
+          echo "show binary logs;" | mysql -uroot -h127.0.0.1 -P3307 -p123456
+          echo "show binary logs;" | mysql -uroot -h127.0.0.1 -P3306 -p123456 | grep -q "mysql-bin.1000003"
+          echo "show binary logs;" | mysql -uroot -h127.0.0.1 -P3307 -p123456 | grep -q "mysql-bin.1000002"
```

Comment:
Mysql 1 trigger 4 times `flush logs` in case and restart 1 time. Mysql 2 trigger 3 times `flush logs` in case and restart 1 time. The origin binlog file is mysql-bin. 999999.